aiogram/bot/base.py

```diff
@@ -190,8 +191,18 @@ async def request(self, method: base.String,
         :rtype: Union[List, Dict]
         :raise: :obj:`aiogram.exceptions.TelegramApiError`
         """
-        return await api.make_request(self.session, self.__token, method, data, files,
-                                      proxy=self.proxy, proxy_auth=self.proxy_auth, timeout=self.timeout, **kwargs)
+        from aiogram import Dispatcher
+        dp = Dispatcher.get_current()
```

Comment:
The bot instance should not know anything about the dispatcher. This change breaks everything.